test/verify/check-services

```diff
@@ -188,6 +189,270 @@ WantedBy=default.target
+        b.click("#boot-or-specific-time .dropdown-menu li[value=2] a")
+        b.click("#drop-repeat button")
+        b.click("#drop-repeat .dropdown-menu li[value=1440] a")
+        b.click("[data-content='add'][data-index=0]")
+        b.set_val("[data-index=0][data-content='hours']", "2")
+        b.set_val("[data-index=0][data-content='minutes']", "40")
+        b.set_val("[data-index=1][data-content='hours']", "21")
+        b.set_val("[data-index=1][data-content='minutes']", "15")
+        b.click("#timer-save-button")
+        b.wait_popdown("timer-dialog")
+        b.wait_present(svc_sel('daily_timer.timer'))
+
+        m.execute("timedatectl set-time '2020-03-17 00:00:00'")
+        m.execute("systemctl start daily_timer.timer")
+        b.wait_present(svc_sel('daily_timer.timer'))
+        b.wait_in_text(svc_sel('daily_timer.timer'), "Daily timer")
+        self.assertIn("Tue 2020-03-17 02:40", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-03-17 03:00:00'")
+        b.wait_present(svc_sel('daily_timer.timer'))
+        self.assertIn("Tue 2020-03-17 21:15", m.execute("LC_ALL=C systemctl list-timers"))
+        # checks if timer runs on friday 2020-04-10 at 02:40 and 21:15
+        m.execute("timedatectl set-time '2020-04-10 00:00:00'")
+        b.wait_present(svc_sel('daily_timer.timer'))
+        self.assertIn("Fri 2020-04-10 02:40", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-04-10 03:00:00'")
+        b.wait_present(svc_sel('daily_timer.timer'))
+        self.assertIn("Fri 2020-04-10 21:15", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("systemctl stop daily_timer.timer")
+        # creates a new houry timer that runs at *:05 and at *:26
+        b.wait_visible("#create-timer")
+        b.click('#create-timer')
+        b.wait_popup("timer-dialog")
+        b.set_val("#servicename", "hourly_timer")
+        b.set_val("#description", "Hourly timer")
+        b.set_val("#command", "/bin/sh -c 'while true; do sleep 5; done'")
+        b.click("#boot-or-specific-time button")
+        b.click("#boot-or-specific-time .dropdown-menu li[value=2] a")
+        b.click("#drop-repeat button")
+        b.click("#drop-repeat .dropdown-menu li[value=60] a")
+        b.click("[data-content='add'][data-index=0]")
+        b.set_val("[data-index=0][data-content='minutes']", "05")
+        b.set_val("[data-index=1][data-content='minutes']", "26")
+        b.click("#timer-save-button")
+        b.wait_popdown("timer-dialog")
+        b.wait_present(svc_sel('hourly_timer.timer'))
+
+        m.execute("timedatectl set-time '2020-04-10 03:00:00'")
+        m.execute("systemctl start hourly_timer.timer")
+        b.wait_present(svc_sel('hourly_timer.timer'))
+        b.wait_in_text(svc_sel('hourly_timer.timer'), "Hourly timer")
+        self.assertIn("Fri 2020-04-10 03:05", m.execute("LC_ALL=C systemctl list-timers"))
```

Comment:
see above, probably a race condition (maybe it would be good to write a function to cover waiting for this info)